modules/zivid/__init__.py

```diff
@@ -24,3 +24,8 @@
 from zivid.settings import (
     Settings,  # noqa: F401 'zivid.settings.Settings' imported but unused
 )
+from zivid.settings_2d import (
+    Settings2d,  # noqa: F401 'zivid.settings.Settings2d' imported but unused
+)
+from zivid.image import Image  # noqa: F401 'zivid.image.Image' imported but unused
+from zivid.frame_2d import Frame2d  # noqa: F401 'zivid.frame_2d.d' imported but unused
```

Comment:
Same

Reply:
My proposal:
zivid.frame_2d: snake_case cant use D, keep as is
Frame2d: CapiTaliZed, D allowed we should use it.